Src/Support/GoogleApis.Auth/OAuth2/Flows/GoogleAuthorizationCodeFlow.cs

```diff
@@ -115,8 +115,13 @@ public new class Initializer : AuthorizationCodeFlow.Initializer
             /// <see cref="Google.Apis.Auth.OAuth2.GoogleAuthConsts.AuthorizationUrl"/>, and Token server URL to 
             /// <see cref="Google.Apis.Auth.OAuth2.GoogleAuthConsts.TokenUrl"/>.
             /// </summary>
-            public Initializer()
-                : base(GoogleAuthConsts.AuthorizationUrl, GoogleAuthConsts.TokenUrl)
+            public Initializer() : this(GoogleAuthConsts.AuthorizationUrl, GoogleAuthConsts.TokenUrl) { }
+
+            /// <summary>Constructs a new initializer.</summary>

```

Comment:
Maybe provide documentation for why this exists? And maybe accept a revoke token URL as well?
(Yes, the property could be set afterwards, but I suspect all three would normally be supplied at once.)